### ISSUE TITLE
MiniProject3 graded before part d is due?

### ISSUE THREAD

**Issue by rwill148** [fdac21/news#11]
Hi Dr. Mockus,

Miniproject3 grades were released in Canvas tonight, but part D is not due until next week. I believe this is why the class average on minirpoject3 is less than a 60%. 

Will the grades for miniproject3 be updated in canvas when part d is done, or is that grade only for parts a-c?

Thanks!

**Comment by audrism** [comment]
Correct, the score is only reflecting a-c with d having twice the weight of others.
